### DISCUSSION TITLE
more tables or more columns?

### DISCUSSION THREAD

**Discussion by hugetiny** [supabase/supabase#12977] [Questions] [answered]
more tables or more columns.
Which way is more efficient in Supabase?

**Comment by GaryAustin1** [comment, marked as answer]
You should just google `postgres more tables or more columns` or something along those lines.  Lots of hits.
For example:
https://www.quora.com/Which-have-a-better-performance-1-table-with-many-columns-or-a-normalized-database-with-several-table
